### PR TITLE
Replace test running inside container with remote variant

### PR DESCRIPTION
**Pull request by erdlet**
As it seems that the test was flaky because of the namespace change from
javax -> jakarta and a update of Arquillian and HTMLUnit didn't help, I
decided to replace the test with a similar one which runs outside the
container. This fixes the build errors on Glassfish and gives us the
same information.

fixes: #276

### CODE REVIEW

File: testsuite/src/main/resources/convert/available/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<!doctype html>
+<html>
+<head>
+    <title>Available converter test</title>
+</head>
+<body>
+    <h1>Available converter test</h1>
+
+    <div id="short">${short}</div>
```
Comment (chkal) [inline testsuite/src/main/resources/convert/available/views/index.jsp:10]: Actually, I'm a bit surprised that this works. Aren't `short`, `double`, etc. reserved keywords? And AFAIK such restrictions apply to EL as well?

**Review by chkal — APPROVED** [review]
Looks good to me! Thanks a lot!

File: testsuite/src/main/resources/convert/available/views/index.jsp (hunk 1 above)
Comment (erdlet) [reply testsuite/src/main/resources/convert/available/views/index.jsp:10]: Hmm good point. I'll change it just to ensure we won't run into any weird behavior later. Thanks for pointing this out.

Comment (chkal) [reply testsuite/src/main/resources/convert/available/views/index.jsp:10]: Awesome! Not sure if this is really an issue TBH. It just came up in my head during the review.

**Review by ivargrimstad — APPROVED** [review]

**Review by ivargrimstad — APPROVED** [review]

File: testsuite/src/main/resources/convert/available/views/index.jsp (hunk 1 above)
Comment (erdlet) [reply testsuite/src/main/resources/convert/available/views/index.jsp]: Done. Now we shouldn't have any problems with that in the future.

**Pull request merged**